lib/util/utils.ts

```diff
@@ -640,9 +632,9 @@ interface Entity {
   type?: DataType
   additionalProperties?: SchemaObject | boolean
   items?: SchemaObject
-  "x-nullable"?: any
+  "x-nullable"?: boolean
```

Comment:
>boolean [](start = 17, length = 7)

can this be ever a string ? I saw lower it its bool or string

Reply:
it can but we convert it to Boolean. We need a good Schema validator :-)